src/server/es/aggs.js

```diff
@@ -554,7 +554,9 @@ export const textAggregation = async (
   let missingAlias = {};
   // don't add missing alias to numeric field by default
   // since the value of missing alias is a string
-  if (config.esConfig.aggregationIncludeMissingData && !isNumericField) {
+  if (config.esConfig.aggregationIncludeMissingData
+    && !isNumericField
+    && config.esConfig.missingDataInFilters) {
```

Comment:
why put this in backend? how is this logic different with just using `aggregationIncludeMissingData`? 

if you put `missingDataInFilters` as `false` in here, it is essentially the same as using `aggregationIncludeMissingData = false` isn;t it? and the charts will also has not missing data in the results returned fropm aggregation

Reply:
thanks, i was so excited to get things working i missed this

I got it working in src/components/ConnectedFilter/utils.js file, how would i read the config flags from this location? or do i need to set a variable in data portal and pass it down? is src/components/ConnectedFilter/utils.js the best location for this functionality?